Code/Mantid/docs/source/algorithms/GenerateEventsFilter-v1.rst

```diff
@@ -81,9 +87,35 @@ this algorithm:
    equal or within the tolerance of the log value as v\_0 + n x delta\_v
    +/- tolerance\_v.
 
+Generate event filters by time
+##############################
+
+Event filters can be created by defining start time, stop time and time intervals. 
+The three input properties for them are *StartTime*, *StopTime* and *TimeInterval*, 
+respectively. 
+
+*TimeInterval* accepts an array of doubles.  
+If the array size is zero, then there will be one and only splitter will be 
+created from *StartTime* and *StopTime*.  
+If the size of the array is one, then all event splitters will have the same duration
+of time. 
+In general if the array is composed as :math:`t_1, t_2, \cdots, t_n`, 
+and :math:`T_0` is the run start time, 
+then the event splitters will have the time boudaries as 
```

Comment:
There's a small typo here: boudaries -> boundaries.

I think this is the type of minor thingy that can be fixed without requiring testing.